content/environment-setup/git/_index.md

```diff
@@ -23,10 +23,13 @@ Git is a critical tool for any kind of coder. We'll get git set up before you do
 
 Start by installing Termux: https://play.google.com/store/apps/details?id=com.termux&hl=en_ZA
 
+Note: Since November 2022, the Google Playstore has refused to accept updates from Termux, so downloading Termux from there won't work. You can download the latest version of Termux from [https://f-droid.org/en/packages/com.termux/](https://f-droid.org/en/packages/com.termux/). F-Droid is a repository of free and open source software for Android devices. You can download the F-Droid app from here [https://f-droid.org/en/](https://f-droid.org/en/).
+
```

Comment:
It may seem as though Termux and F-droid  are the same things, maybe please separate them

start by installing Termux:  https://play.google.com/store/apps/details?id=com.termux&hl=en_ZA

The material regarding the termux update...

Or install F-Droid:  [https://f-droid.org/en/](https://f-droid.org/en/)

Material about F-Droid

This also feels unnessassrily repetitive: `[https://f-droid.org/en/packages/com.termux/](https://f-droid.org/en/packages/com.termux/)`. Maybe say.. `"...[here](https://f-droid.org/en/packages/com.termux/).."`

Reply:
F-droid looks like a place to get the most recent termux update.

I think you should update the section like Sbonelo said, but make sure the learners know that f-droid is not termux, it's an app store to find the latest update